routers/home.go

```diff
@@ -260,16 +263,10 @@ func ExploreOrganizations(ctx *context.Context) {
 	ctx.Data["PageIsExploreOrganizations"] = true
 	ctx.Data["IsRepoIndexerEnabled"] = setting.Indexer.RepoIndexerEnabled
 
-	var ownerID int64
-	if ctx.User != nil && !ctx.User.IsAdmin {
-		ownerID = ctx.User.ID
-	}
-
 	RenderUserSearch(ctx, &models.SearchUserOptions{
 		Type:     models.UserTypeOrganization,
 		PageSize: setting.UI.ExplorePagingNum,
 		Private:  ctx.User != nil,
-		OwnerID:  ownerID,
```

Comment:
why this is removed?

Reply:
because of redundancy: https://github.com/go-gitea/gitea/pull/6274/files#diff-a9097713091436f030e98b5f656d71eeR194